Makefile

```diff
@@ -107,6 +107,12 @@ push: imagetag
 ifeq ($(ARCH),amd64)
 	docker push $(CONTAINER_NAME):$(IMAGETAG)
 	docker push quay.io/$(CONTAINER_NAME):$(IMAGETAG)
+
+	# Push images to gcr.io, used by GKE.
+	gcloud --project projectcalico-org docker -- push gcr.io/projectcalico-org/$(CONTAINER_NAME):$(IMAGETAG)
```

Comment:
I think you should probably push the "-<arch>" suffixed versions too; it'll make the multi-arch migration easier later.

Reply:
@fasaxc done